src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -5862,6 +5890,7 @@ private boolean serverFeaturesModified() {
         if (generateFeatures) {
             // generateFeatures scenario: check if a generated feature has been manually added to other config files
             // Here we pass generated-features.xml instead of server.xml to calculate the generated ones
+            // The second parameter will not be used for generated-features.xml.
```

Comment:
Could you elaborate on this? Just for my understanding.

Reply:
The second parameter (serverDirectory) of getServerXmlFeatures() is only used if the third parameter (generated-features.xml) contains an `<include>` element to bring in more files. Technically the 2nd parameter is used in a logging stmt but that doesn't matter functionally. The point is that it doesn't really matter what value the second parameter has, we don't need to sweat. Unless you're debugging and the log stmt comes out wrong. But our generated file is not going to have an `<include>`.